go.mod

```diff
@@ -62,3 +62,5 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
+
+replace github.com/pingcap/kvproto => /Users/zhangyuanjia/Workspace/go/src/github.com/pingcap/kvproto
```

Comment:
This will be removed after merging https://github.com/pingcap/kvproto/pull/579.

Reply:
You need to replace it with your own repo, otherwise, the CI can not pass.